__tests__/sqlController.js

```diff
@@ -0,0 +1,22 @@
+const { composeSqlQuery } = require('../server/controllers/sqlController');
+
+describe('SQL Query generation unit tests', () => {
+    // TODO: Unique, not null
+    const sampleInput = {
+        table1Name: {
+            column1Name: { primaryKey: true, type: 'varchar' },
+            column2Name: 'varchar',
+            column3Name: 'float',
+            column4Name: 'boolean',
+            column5Name: { linkedTable: 'table2Name' }
+        },
+        table2Name: {
+            column1Name: { primaryKey: true, type: 'integer' },
+            column2Name: 'varchar'
+        }
+    };
```

Comment:
@melissamcl this is the output that I am working off of. Let me know if you have issues with this output!